files/ko/web/javascript/reference/operators/conditional_operator/index.html

```diff
@@ -22,169 +29,68 @@ <h2 id="구문">구문</h2>
 </pre>
 
-<p>또한 다음과 같이 삼항(ternary)의 결과에 따라 변수를 할당할 수도 있다.</p>
-
-<pre class="brush: js notranslate">var elvisLives = Math.PI &gt; 4 ? "Yep" : "Nope";</pre>
-
-<p>다음의 예제처럼, 다중 삼항(ternary) 평가도 가능하다(주의: 조건 연산은 우측부터 그룹핑 됩니다.)</p>
-
-<pre class="brush: js notranslate">var firstCheck = false,
-    secondCheck = false,
-    access = firstCheck ? "Access denied" : secondCheck ? "Access denied" : "Access granted";
-
-console.log( access ); // logs "Access granted"</pre>
-
-<p>또한 다중 조건 IF 문과 같은 방식으로 여러개의 조건을 사용할 수도 있습니다.</p>
-
-<pre class="notranslate"><code>var condition1 = true,
-    condition2 = false,
-    access = condition1 ? (condition2 ? "true true": "true false") : (condition2 ? "false true" : "false false");
-
-console.log(access); // logs "true false"</code></pre>
-
-<p>참고 : 괄호는 필수는 아니며 기능에 영향을주지 않습니다. 결과가 어떻게 처리되는지 시각화하는 데 도움이됩니다.</p>
-
-<p>삼항(ternary) 평가는 다른 연산을 하기 위해 쓸 수도 있습니다.</p>
-
-<pre class="brush: js notranslate">var stop = false, age = 16;
+<h3 id="Handling_null_values">null 값 처리하기</h3>
 
-age &gt; 18 ? location.assign("continue.html") : stop = true;
-</pre>
-
-<p>하나의 케이스 당 둘 이상의 단일 작업을 수행하려면 쉼표로 구분하고 괄호로 묶으세요.</p>
+<p>이와같이 <code>null</code> 값을 처리할 때에도 일반적으로 사용됩니다.:</p>
 
-<pre class="brush: js notranslate">var stop = false, age = 23;
+<pre class="brush: js">let greeting = person =&gt; {
+  let name = person ? person.name : `stranger`
+  return `Howdy, ${name}`
+}
 
-age &gt; 18 ? (
-    alert("OK, you can go."),
-    location.assign("continue.html")
-) : (
-    stop = true,
-    alert("Sorry, you are much too young!")
-);
+console.log(greeting({name: `Alice`}));  // "Howdy, Alice"
+console.log(greeting(null));             // "Howdy, stranger"
 </pre>
 
-<p>또한, 값을 할당하는 동안 하나 이상의 연산도 가능합니다. 이 경우에, 괄호 안의 값중 마지막 쉼표 (,) 다음의 값이 최종 할당 값이 됩니다.</p>
-
-<pre class="brush: js notranslate">var age = 16;
-
-var url = age &gt; 18 ? (
-    alert("OK, you can go."),
-    // alert returns "undefined", but it will be ignored because
-    // isn't the last comma-separated value of the parenthesis
-    "continue.html" // the value to be assigned if age &gt; 18
-) : (
-    alert("You are much too young!"),
-    alert("Sorry :-("),
-    // etc. etc.
-    "stop.html" // the value to be assigned if !(age &gt; 18)
-);
-
-location.assign(url); // "stop.html"</pre>
-
 
+<h3 id="Conditional_chains">연속된 조건문 처리하기</h3>
 
-<h3 id="삼항_연산자로_반환하기">삼항 연산자로 반환하기</h3>
+<p>삼항 연산자는 <code>if … else if … else if … else</code>와 같은 "연속된 조건"을 사용할 수 있습니다.</p>
 
-<p>삼항 연산자는 <code>if / else</code> 문을 사용하는 함수를 반환하는 데 적합합니다.</p>
-
-<pre class="notranslate"><code>var func1 = function( .. ) {
-  if (condition1) { return value1 }
-  else { return value2 }
+<pre class="brush: js">function example(…) {
+    return condition1 ? value1
+         : condition2 ? value2
+         : condition3 ? value3
+         : value4;
 }
 
-var func2 = function( .. ) {
-  return condition1 ? value1 : value2
-}</code></pre>
-
-<p>다음과 같이 법적으로 술을 마실수 있는지 여부를 반환하는 함수를 만들수 있습니다.</p>
+// Equivalent to:
```

Comment:
이 주석은 한글로 변경해도 무방할 것 같아 제안 드립니다!
```suggestion
// 위의 코드는 아래의 코드와 동일합니다.
```